Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLAutoProcess.py

```diff
@@ -116,60 +116,78 @@ def name(self):
         return 'SANSILLAutoProcess'
 
     def validateInputs(self):
+        self.setUp()
+
         result = dict()
-        message = 'Wrong number of {0} runs: {1}. Provide one or as many as sample runs: {2}.'
+        message = 'Wrong number of {0} runs: {1}. Provide one or as many as '\
+                  'sample runs: {2}.'
         message_value = 'Wrong number of {0} values: {1}. Provide one or as ' \
                         'many as sample runs: {2}.'
-        tr_message = 'Wrong number of {0} runs: {1}. Provide one or multiple runs summed with +.'
-        sample_dim = self.getPropertyValue('SampleRuns').count(',')
-        abs_dim = self.getPropertyValue('AbsorberRuns').count(',')
-        beam_dim = self.getPropertyValue('BeamRuns').count(',')
-        flux_dim = self.getPropertyValue('FluxRuns').count(',')
-        can_dim = self.getPropertyValue('ContainerRuns').count(',')
-        str_dim = self.getPropertyValue('SampleTransmissionRuns').count(',')
-        ctr_dim = self.getPropertyValue('ContainerTransmissionRuns').count(',')
-        btr_dim = self.getPropertyValue('TransmissionBeamRuns').count(',')
-        atr_dim = self.getPropertyValue('TransmissionAbsorberRuns').count(',')
-        mask_dim = self.getPropertyValue('MaskFiles').count(',')
-        sens_dim = self.getPropertyValue('SensitivityMaps').count(',')
-        ref_dim = self.getPropertyValue('ReferenceFiles').count(',')
-        maxqxy_dim = self.getPropertyValue('MaxQxy').count(',')
-        deltaq_dim = self.getPropertyValue('DeltaQ').count(',')
-        output_type = self.getPropertyValue('OutputType')
-        n_wedges = self.getProperty('NumberOfWedges').value
+        tr_message = 'Wrong number of {0} runs: {1}. Provide one or multiple ' \
```

Comment:
Are you allowing for more than one transmission in this PR? (i.e. different wavelengths). If not, can you explain what constraints are you imposing on the transmission runs?

Reply:
No, it is still single transmission.
I did not change anything about the constraints on the transmission runs. There should be one transmission only.

Reply:
So if there are many with **,** it will raise an error instead of replacing **,** with **+** ?

Reply:
Yes, a warning message to invite the user to use `+` instead of `,`